acpi_tables/src/aml.rs

```diff
@@ -533,6 +533,36 @@ impl Aml for Io {
     }
 }
 
+pub struct Mmio {
```

Comment:
This is identical to `Memory32Fixed` can you use that instead?

Reply:
Aha, indeed, I added a duplicate struct. Thanks.